### PR TITLE
Generic checkout : Payment method styling, centre checkout container

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

1.  Payment method labels styled, for selected and default.

|before|after|
|-----|-----|
|image|image|

2. Centre Generic Checkout container all breakpoints above mobile

|before|after|
|-----|-----|
|image|image|

[Requirements Document](https://docs.google.com/document/d/1Fey6XoR6nSwpjoh4mRkLTewgMfNfJ9tO3eA8nXU36DQ/edit)

[**Trello Card**](https://trello.com/c/rNXBJ6bo/879-guardian-tier-1-checkout-design-changes)

### CODE REVIEW

File: support-frontend/assets/pages/[countryGroupId]/checkout.tsx [hunk 1]
```diff
@@ -694,6 +706,14 @@ function CheckoutComponent({ geoId }: Props) {
 		}
 	};
 
+	function getLabelText(label: string, checked: boolean): JSX.Element {
+		return (
+			<p css={checked ? checkedRadioLabelColour : defaultRadioLabelColour}>
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/[countryGroupId]/checkout.tsx:711]: Is there a reason we use a `p`? I think this get's wrapped in a `label` in the `Radio` component further down, which feels like odd semantics.

File: support-frontend/assets/pages/[countryGroupId]/checkout.tsx [hunk 2]
```diff
@@ -1165,7 +1185,7 @@ function CheckoutComponent({ geoId }: Props) {
 															]}
 														>
 															<Radio
-																label={label}
+																label={getLabelText(label, selected)}
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/[countryGroupId]/checkout.tsx:1188]: Could we just have this in s ternary here to keep the code a little more succinct?

**Review by jamesgorrie — APPROVED** [review]
Wondering about the semantics of a `p` in a `label` is all. Nice work on getting the centring in too!

File: support-frontend/assets/pages/[countryGroupId]/checkout.tsx (hunk 1 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/[countryGroupId]/checkout.tsx]: No reason, have removed the `p` now, and styled the `label div` directly. Updated.

File: support-frontend/assets/pages/[countryGroupId]/checkout.tsx (hunk 2 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/[countryGroupId]/checkout.tsx]: Yep just doen that in the ocmment above

**Pull request merged**